### PR TITLE
js: Remove JS timeout which got moved to JS context

### PR DESCRIPTION
**Pull request by wallyqs**
Signed-off-by: Waldemar Quevedo <wally@synadia.com>

### CODE REVIEW

File: nats.go [hunk 1]
```diff
@@ -839,14 +834,6 @@ func Timeout(t time.Duration) Option {
 	}
 }
 
-// JetStreamTimeout is an Option to set the timeout for access to the JetStream API
-func JetStreamTimeout(t time.Duration) Option {
```
Comment (derekcollison) [inline nats.go:843]: I use this one, what is it replaced with?

Comment (wallyqs) [reply nats.go:843]: Right now to be able to update the default JS API timeout, it has to be specified when creating the context, not when creating the connection for example:

```go
nc.JetStream(nats.MaxWait(5*time.Second))
```

Comment (derekcollison) [reply nats.go:843]: That is the way I use it.. So that is good.

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**